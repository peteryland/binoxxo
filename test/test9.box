3x3o_x
o6o
o2xx4x
4o_x
o6o
o5o
2xx3x
_o_oo3o
7o
4o
