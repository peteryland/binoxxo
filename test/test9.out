___X___O_X
O______O__
O__XX____X
____O_X___
O______O__
O_____O___
__XX___X__
_O_OO___O_
_______O__
____O_____

XOOXXOXOOX
OXXOOXXOXO
OXOXXOOXOX
XOOXOOXXOX
OXXOXXOOXO
OXOOXXOOXX
XOXXOOXXOO
XOXOOXOXOX
OXOXXOXOXO
XOXOOXOXXO
